Added details for April 'Ask the Brains' event (#338)
asyncjs/async-website: _posts/2023-04-06-ask-the-hybrid-brains-spring-2023.md

---
title: "Ask the Hybrid Brains, Spring 2023"

> summary: "Our Ask the Brains event wherever you are"

Reviewer: ```suggestion
summary: "Ask the Brains is an evening of techy discussion"
```
